src/librustc/hir/lowering.rs

```diff
@@ -95,6 +95,9 @@ pub struct LoweringContext<'a> {
     modules: BTreeMap<NodeId, hir::ModuleItems>,
 
     is_generator: bool,
+    /// This is true iff we are in a function that returns an impl Trait
```

Comment:
nit: iff

Reply:
Ok, I should unabbreviate if-and-only-if.